src/CommunityToolkit.Maui/Alerts/Toast/Toast.shared.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+/// <inheritdoc/>
+public partial class Toast : IToast
```

Comment:
Just like Android, this should be immutable, so we shouldn't implement the `set` for properties.

Reply:
it has methods for it, like this 
![image](https://user-images.githubusercontent.com/33021114/151890465-c0365ba0-e63b-4de3-96c7-5a29a09d7ea8.png)


I prefer to keep properties here.